release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -0,0 +1,1051 @@
+//=== Cloud controller managers for additional cloud providers
+//
+
+[id="ocp-4-16-deprecated-removed-features"]
+== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. For the most recent list of major functionality deprecated and removed within {product-title} {product-version}, refer to the table below. Additional details for more functionality that has been deprecated and removed are listed after the table.
+
+In the following tables, features are marked with the following statuses:
+
+* _General Availability_
+* _Deprecated_
+* _Removed_
+
+[discrete]
+[id="ocp-4-16-operators-dep-rem"]
+=== Operator lifecycle and development deprecated and removed features
+
+.Operator lifecycle and development deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.14 |4.15 |4.16
+
+|SQLite database format for Operator catalogs
+|Deprecated
+|Deprecated
+|Deprecated
+
+|====
+
+[discrete]
+=== Images deprecated and removed features
+
+.Images deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.14 |4.15 |4.16
+
+|`ImageChangesInProgress` condition for Cluster Samples Operator
+|Deprecated
+|Deprecated
+|Deprecated
+
+|`MigrationInProgress` condition for Cluster Samples Operator
+|Deprecated
+|Deprecated
+|Deprecated
+
+|====
+
+[discrete]
+=== Monitoring deprecated and removed features
+
+.Monitoring deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.14 |4.15 |4.16
+
+|`dedicatedServiceMonitors` setting that enables dedicated service monitors for core platform monitoring
+|General Availability
+|Deprecated
+|Deprecated
+
+|====
+
+[discrete]
+=== Installation deprecated and removed features
+
+.Installation deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.14 |4.15 |4.16
+
+|OpenShift SDN network plugin
+|Deprecated
+|Removed ^[1]^
+|Removed
+
+|`--cloud` parameter for `oc adm release extract`
+|Deprecated
+|Deprecated
+|Deprecated
+
+|CoreDNS wildcard queries for the `cluster.local` domain
+|Deprecated
+|Deprecated
+|Deprecated
+
+|`compute.platform.openstack.rootVolume.type` for {rh-openstack}
+|Deprecated
+|Deprecated
+|Deprecated
+
+|`controlPlane.platform.openstack.rootVolume.type` for {rh-openstack}
+|Deprecated
+|Deprecated
+|Deprecated
+
+|`ingressVIP` and `apiVIP` settings in the `install-config.yaml` file for installer-provisioned infrastructure clusters
+|Deprecated
+|Deprecated
+|Deprecated
+
+|====
+[.small]
+--
+1. While the OpenShift SDN network plugin is no longer supported by the installation program in version 4.15, you can upgrade a cluster that uses the OpenShift SDN plugin from version 4.14 to version 4.15.
+--
+[discrete]
```

Comment:
can remove "Updating clusters dep and rem features because nothing there or comment it out